lib/components/stream_preview.dart

```diff
@@ -76,17 +76,17 @@ class _StreamPreviewState extends State<StreamPreview> {
               name: "Flutter",
               onMessageReceived: (message) {
                 final params = jsonDecode(message.message)["params"];
-                if (params != null) {
+                if (params is Map) {
                   setState(() => _playerState = params["playback"]);
```

Comment:
Not exactly from this PR, but it looks like this `setState((){})` will throw errors as it's sometimes called then the widget is not mounted. It does not seem to crash the app.

Reply:
Oh good catch, I added some mounted guards to catch that.